packages/widget_toolkit_biometrics/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: widget_toolkit_biometrics
 description: A Flutter package that helps users to handle enabling and disabling biometrics in an application
-version: 0.0.2
+version: 0.0.3
```

Comment:
Since we are updating the `rxdart` and `rx_bloc` to the next major version, we should treat this as a major version because of potential incompatibilities when integrating in other projects. So we can update this to `0.1.0`.

Make sure to update the changelog too.

Reply:
Good catch, I didn't know this. I've updated them